### PR TITLE
Rule Edit Modal

### PR DESCRIPTION
**Pull request by Trishthedish**
Again, this is work has already been merged with master but encapsulates the changes made to get my edit rule functionality to work. 

I continued to work on functionality after this work was completed.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,10 +3,10 @@ import logo from './logo.svg';
 import './App.css';
 import 'react-virtualized/styles.css'
 // Using an ES6 transpiler like Babel
-import {SortableContainer, SortableElement, arrayMove} from 'react-sortable-hoc';
+// import {SortableContainer, SortableElement, arrayMove} from 'react-sortable-hoc';
```
Comment (tony-bye) [inline src/App.js:6]: Please delete any commented out code, like these imports.

File: src/App.js [hunk 2]
```diff
@@ -47,9 +47,9 @@ const store = configureStore()
 //   height: 40,
 // };
 
-const buttonStyle = {
-  margin: 12,
-}
+// const buttonStyle = {
```
Comment (tony-bye) [inline src/App.js:50]: And delete any commented out css like this.

**Review by tony-bye — APPROVED** [review]
Good

**Comment by TheDahv** [comment]
@Trishthedish could you please close this PR or remove me from reviewers?

**Comment by tony-bye** [comment]
According to description, this was already merged.

**Pull request closed without merging**